src/main/resources/site/parts/highmap/Highmap.jsx

```diff
@@ -167,13 +152,22 @@ function Highmap(props) {
   }
 
   return (
-    <section className='part-highmap container'>
+    <section className='xp-part part-highchart'>
       <Row>
-        <Col className='col-12 p-lg-0'>
-          <HighchartsReact highcharts={Highcharts} constructorType={'mapChart'} options={mapOptions} />
+        <Col className='col-12'>
+          <figure className='highcharts-figure mb-0 hide-title'>
+            {mapOptions.title?.text && <figcaption className='figure-title'>{mapOptions.title.text}</figcaption>}
+            {mapOptions.subtitle?.text && <p className='figure-subtitle'>{mapOptions.subtitle.text}</p>}
```

Comment:
Synes det ser ut som tittel/undertittel legges på på en helt annen måte enn det gjøres i highcharts. Siden vi ønsker å bruke samme styling, bør det ikke gjøres på samme måte?

(Bildet viser jsx for highcharts vs mighmaps)
![Screenshot 2024-02-09 at 14 40 59](https://github.com/statisticsnorway/mimir/assets/142512638/3c46ba5c-9bf8-4b08-9ef3-42d054bf6e8d)

Reply:
Ah, stemmer det. Men da ser det likt ut 👍